new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleTraefikValidation.java

```diff
@@ -0,0 +1,726 @@
+            .helmParams(opHelmParams)
+            .image(OPERATOR_IMAGE)
+            .domainNamespaces(Arrays.asList(domainNamespace))
+            .serviceAccount(serviceAccountName);
+
+    // install Operator
+    logger.info("Installing Operator in namespace {0}", opNamespace);
+    assertTrue(installOperator(opParams),
+        String.format("Operator install failed in namespace %s", opNamespace));
+    logger.info("Operator installed in namespace {0}", opNamespace);
+
+    // list helm releases
+    logger.info("List helm releases in namespace {0}", opNamespace);
+    String cmd = String.format("helm list -n %s", opNamespace);
+    assertTrue(getExpectedResult(cmd, OPERATOR_RELEASE_NAME),
+        String.format("did not get %1s in namespace %2s", OPERATOR_RELEASE_NAME, opNamespace));
+
+    // check operator is running
+    logger.info("Check Operator pod is running in namespace {0}", opNamespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for operator to be running in namespace {0} "
+                    + "(elapsed time {1}ms, remaining time {2}ms)",
+                opNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(operatorIsRunning(opNamespace));
+  }
+
+  /**
+   * Install traefik and wait until the traefik pod is ready.
+   */
+  private static void installAndVerifyTraefik() {
+
+    // create a new unique namespace for traefik
+    logger.info("Creating an unique namespace for Traefil");
+    tfNamespace = assertDoesNotThrow(() -> createUniqueNamespace(),
+        "Fail to create an unique namespace due to ApiException");
+    logger.info("Created a new namespace called {0}", tfNamespace);
+
+    // helm install parameters
+    tfHelmParams = new HelmParams()
+        .releaseName("traefik-operator")
+        .namespace(tfNamespace)
+        .repoUrl("https://kubernetes-charts.storage.googleapis.com/")
+        .repoName("stable")
+        .chartName("traefik")
+        .chartValuesFile(projectRoot + "/../kubernetes/samples/charts/traefik/values.yaml");
+
+    TraefikParams tfParams = new TraefikParams()
+        .helmParams(tfHelmParams)
+        .namespaces("{" + tfNamespace + "," + domainNamespace + "}");
+
+    // install Traefik
+    assertThat(installTraefik(tfParams))
+        .as("Test installTraefik returns true")
+        .withFailMessage("installTraefik() did not return true")
+        .isTrue();
+    logger.info("Traefik is installed in namespace {0}", tfNamespace);
+
+    // verify that the traefik is installed
+    String cmd = "helm ls -n " + tfNamespace;
+    assertThat(getExpectedResult(cmd, tfHelmParams.getReleaseName()))
+        .as("traefik is installed")
+        .withFailMessage("did not find traefik-operator in " + tfNamespace)
+        .isTrue();
+
+    // first wait 30 seconds, then check if the traefik pod is ready.
+    with().pollDelay(30, SECONDS)
```

Comment:
don't have to wait for 30 seconds, change delay seconds to 5

Reply:
changed to 5 seconds